index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/styles/styles.css">
+  <link rel="stylesheet" href="/about.html">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:wght@300&display=swap"
+    rel="stylesheet">
+  <title>Mike's Capstone Project</title>
+</head>
+
+<body>
+  <!-- the navbar (and header) section -->
+  <nav class="mini-nav on-desktop">
+    <ul class="mininav-menu">
+      <li class="mininav-item"><img src="/img/facebook.svg" alt="facebook"></li>
+      <li class="mininav-item"><img src="/img/twitter.svg" alt="twitter"></li>
+      <li class="mininav-menu">English</li>
+      <li class="mininav-menu">My Page</li>
+      <li class="mininav-menu">Logout</li>
+    </ul>
+  </nav>
+  <header id="myheader">
+    <!-- <a class="logo" href="#home">My Logo</a> -->
+    <a class=" logo on-desktop" href="#home"><img src="/img/logo.jpg" class="logo-nav" alt="logo"></a>
```

Comment:
- [ ] please kindly align your logo with the text in the main section like in the original design.
- [ ] kindly add the font size of the main heading and make it bolder as well
- [ ] please add some padding top and bottom below the date and above the main heading, so the main section can follow the design.
yours
![Screenshot (1075)_LI](https://github.com/Mike111222/CapstoneProject/assets/65298048/1a011882-40f7-4b4d-a472-04e119a67278)
original
![Screenshot (1071)](https://github.com/Mike111222/CapstoneProject/assets/65298048/51792ea6-d9de-4f9d-b8d4-90d9b560db45)

Reply:
I have implemented all of the above-mentioned changes. Thanks a lot again